modules/ROOT/pages/rtc/rtc-troubleshooting.adoc

```diff
@@ -66,4 +66,4 @@ A number of part-related errors can be raised if the JWT fails to decode:
 * A part was not correctly base64 encoded
 * A part was not valid JSON
 * The header part does not specify an algorithm
-* The payload part does not contain all link:{baseurl}/rtc/jwt-authentication/#claims[required claims]
+* The payload part does not contain all xref:rtc/jwt-authentication.adoc#claims[required claims]
```

Comment:
This xref doesn't resolve since the `claims` id doesn't exist. It's broken on production as well, however I believe the heading got renamed at one point though and as such it should probably be using this:
```suggestion
* The payload part does not contain all xref:rtc/jwt-authentication.adoc#requiredjwtclaimsforreal-timecollaboration[required claims]
```

Reply:
ok, I changed it